webapp/channels/src/components/widgets/modals/components/save_changes_panel.scss

```diff
@@ -1,68 +1,128 @@
 .mm-save-changes-panel {
-    position: absolute;
+    position: fixed;
     z-index: 1000;
-    right: 20px;
-    bottom: 20px;
     display: flex;
-    width: calc(100% - 232px - 40px);
+    width: calc(63%);
     align-items: center;
-    padding: 18px;
-    background-color: var(--dnd-indicator);
+    padding: 12px;
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+    animation-duration: 300ms;
+    animation-fill-mode: forwards;
+    animation-name: entry-animation;
+    animation-timing-function: ease;
+    background: var(--center-channel-bg);
     border-radius: 4px;
     box-shadow: var(--elevation-3);
     font-family: 'Open Sans', sans-serif;
+    transition: all 300ms ease;
+
+    @keyframes entry-animation {
+        from {
+            bottom: 0;
+        }
+
+        to {
+            bottom: 20px;
+        }
+    }
+
+    @media screen and (max-width: 768px) {
+        width: calc(95%);
+    }
+
+    &.error {
+        border-color: var(--dnd-indicator);
+        background: var(--dnd-indicator);
+    }
+
+    &.saved {
+        border-color: var(--online-indicator);
+        background: var(--online-indicator);
+    }
 
     &__message {
         display: flex;
         align-items: center;
         margin: 0;
-        color: var(--sidebar-text);
+        color: var(--center-channel-color);
         font-family: inherit;
         font-size: 14px;
         font-style: normal;
         font-weight: 600;
         line-height: 20px;
+
+        span {
+            margin-left: 8px;
+        }
+
+        svg {
+            color: rgba(var(--center-channel-color-rgb), 0.56);
+        }
+
+        &.error,
+        &.saved {
+            color: var(--button-color);
+
+            svg {
+                color: var(--button-color);
+            }
+        }
     }
 
     &__btn-ctr {
         display: flex;
         flex: 1 1 auto;
         justify-content: flex-end;
         gap: 8px;
+
+        #panelCloseButton {
+            color: rgba(var(--button-color-rgb), 0.64);
+            font-size: 16px;
+        }
     }
 
     &__cancel-btn {
         display: flex;
-        width: 64px;
+        min-width: 64px;
         height: 32px;
         align-items: center;
         padding: 10px 16px;
         border: none;
-        background: rgba(255, 255, 255, 0.12);
+        background: rgba(var(--denim-button-bg-rgb), 0.08);
```

Comment:
```suggestion
        background: rgba(var(--button-bg-rgb), 0.08);
```
@sinansonmez I think you shouldn't use a denim theme colour directly 
Theming will not work properly if you use these see the below

https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/sass/base/_css_variables.scss

https://github.com/mattermost/mattermost/blob/61b1f2641184646b81334f690a49bdbdc7211347/webapp/channels/src/packages/mattermost-redux/src/selectors/entities/preferences.ts#L116

<img width="351" alt="Screenshot 2024-01-30 at 12 01 18 PM" src="https://github.com/mattermost/mattermost/assets/16203333/0dd57a8f-6989-42d4-a947-e440be0743f6">
<img width="550" alt="Screenshot 2024-01-30 at 12 00 37 PM" src="https://github.com/mattermost/mattermost/assets/16203333/166ca627-5711-41ed-bfb1-d85441927f6e">
<img width="546" alt="Screenshot 2024-01-30 at 11 58 56 AM" src="https://github.com/mattermost/mattermost/assets/16203333/f4a9aa0c-e15d-4b7f-8662-020d616dda5c">



cc// @hmhealey

Reply:
Good catch. I still haven't figured out exactly why some designs reference the denim versions of those variables specifically

Reply:
Good catch @AshishDhama I usually forget this rule, when I see `denim` in Figma file. I will review all.